src/apis/api.ts

```diff
@@ -0,0 +1,41 @@
+import { getLocalStorge } from '@/utils/util';
+import axios, { AxiosInstance } from 'axios';
+
+const TIME_OUT = 10000;
```

Comment:
요렇게 표현해줘도 괜찮을 것 같네요~
```suggestion
const TIME_OUT = 10_000;
```

Reply:
10초는 너무 길게 느껴지는데 어떠신가요?

Reply:
사용자 입장에서 생각하면 늦어질수 있다고 느껴지네요 
우선 5초로 줄이겠습니다